globals.go

```diff
@@ -291,6 +291,13 @@ var (
 		RepotrackerVersionRequester,
 		TriggerRequester,
 	}
+
+	ProviderEc2Type = []string{
+		ProviderNameEc2Auto,
+		ProviderNameEc2OnDemand,
```

Comment:
do we need to run the check on OnDemand hosts?

Reply:
Fixed.